docs/developer-center/guides/06-Explore-the-Data-Observatory-catalog.md

```diff
@@ -0,0 +1,5614 @@
+    </tr>
+    <tr>
+      <th>3</th>
+      <td>[bq]</td>
+      <td>demographics</td>
+      <td>Demographics</td>
+      <td>usa</td>
+      <td>consumerprofiles</td>
+      <td>Segmentation of the population in sixty-eight ...</td>
+      <td>None</td>
+      <td>carto-do.ags.geography_usa_blockgroup_2015</td>
+      <td>USA Census Block Group</td>
+      <td>carto-do.ags.demographics_consumerprofiles_usa...</td>
+      <td>...</td>
+      <td>eng</td>
+      <td>Consumer Profiles</td>
+      <td>ags</td>
+      <td>Applied Geographic Solutions</td>
+      <td>ags_consumerpro_e8344e2e</td>
+      <td>{'counts': {'rows': 217182, 'cells': 31057026,...</td>
+      <td>yearly</td>
+      <td>[2018-01-01,2019-01-01)</td>
+      <td>None</td>
+      <td>2018</td>
+    </tr>
+    <tr>
+      <th>4</th>
+      <td>[bq]</td>
+      <td>demographics</td>
+      <td>Demographics</td>
+      <td>usa</td>
+      <td>businesscounts</td>
+      <td>Business Counts database is a geographic summa...</td>
+      <td>None</td>
+      <td>carto-do.ags.geography_usa_blockgroup_2015</td>
+      <td>USA Census Block Group</td>
+      <td>carto-do.ags.demographics_businesscounts_usa_b...</td>
+      <td>...</td>
+      <td>eng</td>
+      <td>Business Counts</td>
+      <td>ags</td>
+      <td>Applied Geographic Solutions</td>
+      <td>ags_businesscou_a8310a11</td>
+      <td>{'counts': {'rows': 217182, 'cells': 25627476,...</td>
+      <td>yearly</td>
+      <td>[2018-01-01,2019-01-01)</td>
+      <td>None</td>
+      <td>2018</td>
+    </tr>
+    <tr>
+      <th>5</th>
+      <td>[bq]</td>
+      <td>demographics</td>
+      <td>Demographics</td>
+      <td>usa</td>
+      <td>crimerisk</td>
+      <td>Using advanced statistical methodologies and a...</td>
+      <td>None</td>
+      <td>carto-do.ags.geography_usa_blockgroup_2015</td>
+      <td>USA Census Block Group</td>
+      <td>carto-do.ags.demographics_crimerisk_usa_blockg...</td>
+      <td>...</td>
+      <td>eng</td>
+      <td>Crime Risk</td>
+      <td>ags</td>
+      <td>Applied Geographic Solutions</td>
+      <td>ags_crimerisk_9ec89442</td>
+      <td>{'counts': {'rows': 217182, 'cells': 3040548, ...</td>
+      <td>yearly</td>
+      <td>[2018-01-01,2019-01-01)</td>
+      <td>None</td>
+      <td>2018</td>
+    </tr>
+  </tbody>
+</table>
+<p>6 rows × 21 columns</p>
+</div>
+
+
+
+They comprise different information: consumer spending, retail potential, consumer profiles, etc.
+
+At a first sight, it looks the dataset with `data_source_id: sociodemographic` might contain the population information we are looking for. Let's try to understand a little bit better what data this dataset contains by looking at its variables:
+
+
+```python
+from cartoframes.data.observatory import Dataset
+dataset = Dataset.get('ags_sociodemogr_e92b1637')
+variables = dataset.variables
+variables
+```
+
+
+
+
+    [<Variable.get('HINCYMED65_310bc888')> #'Median Household Income: Age 65-74 (2019A)',
```

Comment:
This piece of code is also quite long. Do you think it'd be ok to  ✂️cut  ✂️it a bit, and have something like:

```
[<Variable.get('HINCYMED55_1a269b4b')> #'Median Household Income: Age 55-64 (2019A)',
<Variable.get('HINCYMED55_1a269b4b')> #'Median Household Income: Age 55-64 (2019A)',
<Variable.get('HINCYMED55_1a269b4b')> #'Median Household Income: Age 55-64 (2019A)',
...
<Variable.get('HINCYMED55_1a269b4b')> #'Median Household Income: Age 55-64 (2019A)',
<Variable.get('HINCYMED55_1a269b4b')> #'Median Household Income: Age 55-64 (2019A)',
<Variable.get('HINCYMED55_1a269b4b')> #'Median Household Income: Age 55-64 (2019A)',]
```

Another option would be to put this inside a `<div>` with the `u-vertical-scroll` class (the same we'll be using in the tables), but that might require more adjustments.

Reply:
yep, I prefer the scroll thing, I'll add it. Thanks!